crates/kas-widgets/src/filler.rs

```diff
@@ -14,7 +14,7 @@ mod Filler {
     /// This widget has zero minimum size but can expand according to the given
     /// stretch priority.
     #[derive(Clone, Debug, Default)]
-    #[widget(type Data = ())]
+    #[widget]
     pub struct Filler {
```

Comment:
No `Data` def needed here now. Since `Data` cannot be used anywhere, the restriction to `()` is unambiguous.